js/buda.js

```diff
@@ -558,21 +558,22 @@ module.exports = class buda extends Exchange {
         //
         const id = this.safeString (order, 'id');
         const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
+        const datetime = this.iso8601 (timestamp);
         const marketId = this.safeString (order, 'market_id');
         const symbol = this.safeSymbol (marketId, market, '-');
         const type = this.safeString (order, 'price_type');
         const side = this.safeStringLower (order, 'type');
         const status = this.parseOrderStatus (this.safeString (order, 'state'));
         const originalAmount = this.safeValue (order, 'original_amount', []);
-        const amount = this.safeNumber (originalAmount, 0);
+        const amount = this.safeString (originalAmount, 0);
```

Comment:
In cases like these, the default fallback-values must be numeric strings like `'0'`.